pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -2336,36 +2362,34 @@ func backoffTesterForPublish(t *testing.T, config *backoffTesterConfig) {
 
 	// Mock clock tick for the 200th millisecond. So backoff condition is no longer true.
 	tc.Step(step)
-	runPublishRequest := func(req *csi.ControllerPublishVolumeRequest) <-chan error {
+	runPublishRequest := func(req *csi.ControllerPublishVolumeRequest, reportError bool) error {
 		response := make(chan error)
 		go func() {
 			_, err := driver.cs.ControllerPublishVolume(context.Background(), req)
 			response <- err
 		}()
-		return response
+		go func() {
+			<-readyToExecute <- gcecloudprovider.Signal{ReportError: reportError}
```

Comment:
the ordering here is a bit difficult to understand. Can we keep the multi-line read from channel in this go routine for readability?
```
	execute := <-readyToExecute
	s1 := gcecloudprovider.Signal{ReportError: true}
	execute <- s1
```

Reply:
done